pkg/box/box.go

```diff
@@ -56,6 +56,8 @@ type OktaConfig struct {
 	OIDCClientID string `yaml:"oidcClientID"`
 	// OrgDomain is the hostname of the Okta instance.
 	OrgDomain string `yaml:"orgDomain"`
+	// SessionDuration is the TTL for the session.
```

Comment:
suggestion: always good to note what units a number represents
```suggestion
	// SessionDuration is the TTL for the session in seconds.
```